app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Reportback;
+use League\Fractal\TransformerAbstract;
+
+class ReportbackTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param User $user
+     * @return array
+     */
+    public function transform(Reportback $reportback)
+    {
+        return [
+            'id' => (string) $reportback->id,
```

Comment:
I think these would make sense to handle with [attribute casting](https://laravel.com/docs/5.2/eloquent-mutators#attribute-casting) on the model itself, so we don't have to remember to cast them everywhere we use in the codebase.